src/python_redlines/api.py

```
@@ -0,0 +1,126 @@
+    )
+
+    return Response(
+        content=redline_output,
+        media_type="application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+        headers={"Content-Disposition": "attachment; filename=redline.docx"},
+    )
+
+
+@app.post("/redline/double", dependencies=[Depends(verify_token)])
+async def redline_double(
+    original: UploadFile = File(...),
```

Ruff (0.13.3)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)